pkg/scheduler/internal/cache/cache_test.go

```diff
@@ -110,10 +110,12 @@ func TestAssumePodScheduled(t *testing.T) {
 	}
```

Comment:
Could you have name each of `testPods`> something that we can easily understand how the Pod looks like from the name.

Reply:
The third parameter `objName` in the `makeBasePod` function is the name of the passed pod. I think it can be modified to a name that is easier to understand the purpose of the test, instead of just "test" and "test-1", is that OK?

Reply:
Update the pod name, please review it again.